libs/testing/containers/src/lib/testing-containers.ts

```diff
@@ -0,0 +1,39 @@
+import { GenericContainer, StartedTestContainer, Wait } from 'testcontainers'
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace NodeJS {
+    interface Global {
+      __CONTAINER_POSTGRES__: StartedTestContainer
```

Comment:
Please enlighten me, what is this beauty for? 🧐

Reply:
To be able to access the container created in globalSetup in the globalTeardown function, I save a reference to the container in the global scope. 
[https://jestjs.io/docs/configuration#globalsetup-string](https://jestjs.io/docs/configuration#globalsetup-string)

Reply:
As I write this I realize it is unnecessary to have it in the global scope. I can just place it at the top of this file as the variable is only used in this file.